stable/postgresql/templates/secrets.yaml

```diff
@@ -13,4 +13,7 @@ data:
   postgres-password:  {{ .Values.postgresPassword | b64enc | quote }}
   {{ else }}
   postgres-password: {{ randAlphaNum 10 | b64enc | quote }}
-  {{ end }}
\ No newline at end of file
+  {{ end }}
+  {{- if .Values.metrics.customMetrics }}
+  custom-metrics.yaml: {{ toYaml .Values.metrics.customMetrics | b64enc | quote }}
```

Comment:
I thought about that too. I decided against it because conditionally creating another resource and adding a second mount to the deployment seemed to much complexity so i piggybacked on the already exiting secret.
That being said, I can change it if desired.